Gruntfile.js

```diff
@@ -45,16 +45,28 @@ module.exports = function(grunt) {
 
   var env = grunt.option("env") || "release";
 
+  var pkg = grunt.file.readJSON("package.json");
+
+  var optionsPath;
+
+  if (fs.existsSync("options")) {
+    optionsPath = "options";
+  }
+  else if (fs.existsSync("node_modules/worldview-options-eosdis")) {
```

Comment:
Lines 50-58 could just be;
`var optionsPath = hasOptionsDirectory? 'options' : 'node_modules/worldview-options-eosdis';`

And I'd move that up to the top of the file right after setting `hasOptionsDirectory`.